test/dlc_tests/container_tests/bin/healthcheck_tests/efa_checker_test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+echo "Start EFA checker test"
+efa_checker_single_node --no_instance_id --verbose
+echo "Complete EFA checker test"
```

Comment:
These scripts will always give a successful execution (exit code) even when `efa_checker_single_node --no_instance_id --verbose` command fails. We would need to ensure that we return an appropriate exit code in case of a failure. 

Please refer to this: https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/container_tests/bin/dgl_tests/testDGLHelper

Reply:
synced up offline, 'set -e' already set the exit code when there is an error